### PR TITLE
14 issue Add documentation using mkdocs

### PR DESCRIPTION
**Pull request by naz-hage**

### CODE REVIEW

File: docs/usage.md [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+
+```csharp
+using Ntools;
+
+var isElevated = CurrentProcess.IsElevated();
+Console.WriteLine(isElevated);
+```
+
+
+**Boost your productivity with `ntools-launcher`**, an efficient library for automating common development tasks.
```
Comment (ahmdhjj) [inline docs/usage.md:106]: This feels out of place in this page. Is it meant for introduction?

File: docs/usage.md [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+**Key benefits:**
+
+* **Improved Efficiency:** Automate repetitive tasks and focus on core development activities.
+* **Simplified Workflow:** User-friendly API streamlines integration into your projects.
+* **Enhanced Control:** Manage processes and ensure proper elevation levels for secure development.
+
+Start saving time and effort today! Install `ntools-launcher` via NuGet.
+
+The library is used by the [ntools's](https://github.com/naz-hage/ntools) repo
+- **Nbackup** launches `robocopy` and wait for it to complete, ensuring that the backup process is fully completed before the program continues.
```
Comment (ahmdhjj) [inline docs/usage.md:124]: This part is not properly formatted

**Review by ahmdhjj — CHANGES_REQUESTED** [review]

File: docs/revisions/changelog.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
+### Version 1.6.0 - 07-may-24
+- [issue #14](https://github.com/naz-hage/home/issues/14) - Feature: Publish documentation using mkdocs
+
+
+### Version 1.5.0 - 29-feb-24
+- Add DownloadAsync method to download file from the web.
+    - The method uses the VirusTotal API to check the downloaded file for virus.
+    - The method returns a ResultDownload object that details the download result such as file signature, file size, and VirusTotal check.
+
+    [Next](#next)
```
[the remark this answers is not in the document]
Comment (naz-hage) [reply docs/revisions/changelog.md:10]: Good catch! This was a leftover text.

File: docs/installation.md [hunk 4]
```diff
@@ -0,0 +1,4 @@
+Installation
+The ntools-launcher package is available on nuget.org. To install the package, run the following command in the Package Manager Console:
+
+Install-Package ntools-launcher
```
[the remark this answers is not in the document]
Comment (naz-hage) [reply docs/installation.md:4]: Fixed format and added a few more details.

File: docs/usage.md (hunk 1 above)
Comment (naz-hage) [reply docs/usage.md:106]: Yes, This should move to the introduction.

File: docs/usage.md (hunk 2 above)
Comment (naz-hage) [reply docs/usage.md:124]: Formatted it correctly.

**Review by naz-hage — COMMENTED** [review]
Thanks for your comments @ahmdhjj. I resolved all issues.

**Review by ahmdhjj — APPROVED** [review]

**Pull request merged**